Sources/TuistSupport/Utils/DeveloperEnvironment.swift

```diff
@@ -10,6 +10,8 @@ public protocol DeveloperEnvironmenting {
 }
 
 public final class DeveloperEnvironment: DeveloperEnvironmenting {
```

Comment:
I'd merge `DeveloperEnvironment` into `Environment`, since we don't take any advantage of this distinction, but I'd do that in a follow-up PR.